app/d3plus.css

```diff
@@ -7,35 +7,77 @@
 
 .d3plus-tooltip {
   box-shadow: rgba(0, 0, 0, 0.25) 0px 1px 3px;
-  max-width: 225px;
-  min-width: 150px;
+  max-width: 224px;
+  min-width: 224px;
   z-index: 30;
+  padding: 0px !important;
+  border-radius: 6px !important;
```

Comment:
Both of these can be defined in `tooltipConfig`, so we shouldn't need `!important` overrides in CSS. Here is a list of all of the config methods that can be set in `d3plus.js`: https://github.com/d3plus/d3plus-tooltip#api-reference

If they _can_ be set in JavaScript, we _should_ set them in JavaScript.